kuma/static/styles/components/wiki/indicators.styl

```diff
@@ -134,7 +134,8 @@ icons for both types of indicators
 .experimental,
 .experimentalBadge,
 /* es7 and harmony macros  */
-.overheadIndicator[style='background: #9CF49C;'] {
+.overheadIndicator[style='background: #9CF49C;'],
+.overheadIndicator[style='background: rgb(156, 244, 156);'] // IE10 {
```

Comment:
Opening curly brace got caught in the comment, and commas in the `rgb` function need to be escaped for Stylus to compile properly.